docs/manual/forms/cosign_flow.rst

```diff
@@ -0,0 +1,32 @@
+.. _manual_cosign_flow:
+
+Mede-ondertekenen
+=================
+
+Flow
+----
+
+De mede-ondertekenen flow is hieronder beschreven. Persoon 1 is de persoon die als eerste het formulier invult.
+Persoon 2 is de mede-ondertekenaar.
+
+#. Persoon 1 logt in om het formulier te beginnen.
+#. In het formulier bevindt zich een "Mede-ondertekenen" component. Hier kan persoon 1 de email adres invoeren
+   van persoon 2.
+#. Op het moment dat het formulier wordt ingestuurd, krijgt persoon 1 een bevestigingsemail (als dit
+   geconfigureerd is in het formulier).
```

Comment:
```suggestion
#. Zodra het formulier ingestuurd wordt, krijgt persoon 1 een bevestigingsmail (als deze ingeschakeld is).
```
"bevestigingsmail" or "bevestigings-e-mail" are both correct spellings, I prefer the former because the latter looks weird.